requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
No. I don't know of any such convention. The closest I've read about is ``test_requires`` argument for when you run ``python setup.py test``, right?

I usually separate dependencies into 1. general pip, 2. conda (FTW), 3. development. But then just install them for development using ``conda install --file requirements/conda.txt && pip install -r requirements/development.txt``.

Reference: https://github.com/robinandeer/chanjo/tree/master/requirements